ext/opentelemetry-ext-system-metrics/tests/test_system_metrics.py

```diff
@@ -0,0 +1,212 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import namedtuple
+from unittest import mock
+
+from opentelemetry import metrics
+from opentelemetry.ext.system_metrics import SystemMetrics
+from opentelemetry.sdk.metrics import Observer
+from opentelemetry.test.test_base import TestBase
+
+
+class TestSystemMetrics(TestBase):
```

Comment:
These tests are really nice and cover the different units. I'm wondering if a test that creates a `SystemMetrics` using an in-memory exporter that checks if metrics are generated is missing.

Reply:
Yes, I agree. I'll try to add this today.

Reply:
Done